vaadin-grid-flow-demo/src/main/java/com/vaadin/flow/component/treegrid/demo/data/DepartmentData.java

```diff
@@ -8,7 +8,7 @@
 import java.util.stream.Collectors;
 
 public class DepartmentData {
-    private static final List<Department> DEPARTMENT_LIST = createDepartmentList();
+    private final List<Department> DEPARTMENT_LIST = createDepartmentList();
```

Comment:
Why remove 'static'?

Reply:
Previously, the grid was read-only, but now it is not. The parent of the dragged items is changing by dragging and dropping items.